python/tank/platform/engine.py

```diff
@@ -192,6 +194,10 @@ def __init__(self, tk, context, engine_instance_name, env):
         qt_abstraction.QtCore = qt.QtCore
         qt_abstraction.QtGui = qt.QtGui
 
+        # load the fonts. this will work if there is a QApplication instance
+        # available.
+        self._ensure_core_fonts_loaded()
```

Comment:
@manneohrstrom I've moved the `automethod` calls out of here and into the `platform.rst` file. I've included the `_ensure_core_font_loaded()` method in there and put all the methods into an **Engine Customizations** group. 

 Just a heads up to make sure you're aware of it.